Add episode 55 (#608)
trinodb/trino.io: _episodes/55.md

  at [Starburst](https://starburst.io)
* Manfred Moser, Director of Technical Content at
  [Starburst](https://starburst.io)
  ([@simpligility](https://twitter.com/simpligility))

## Guests

* [Mustafa Sakalsiz](https://linkedin.com/in/sakalsiz), CEO at
  [Peaka](https://www.peaka.com/)
* [Ali Tekin](https://www.linkedin.com/in/alitekin/), Principal Software
  Architect at [Peaka](https://www.peaka.com/)

## Releases 437-438

[Trino 437](https://trino.io/docs/current/release/release-437.html)

* Support for configuring compression codecs
* Support for `char` values in the `to_utf8()` and `lpad()` functions
* Improved performance for Delta Lake queries without table statistics
* Improved performance for Iceberg queries with filters on `ROW` columns

[Trino 438](https://trino.io/docs/current/release/release-438.html)

* Support for access control with [Open Policy Agent](https://trino.io/blog/2024/02/06/opa-arrived)
* Support for `ALTER COLUMN ... DROP NOT NULL` in Iceberg and PostgreSQL
* Support for configuring page sizes in Delta Lake, Hive, and Iceberg
* Better type support for the `reduce_agg()` function

> * [Trino Gateway version 5](https://github.com/trinodb/trino-gateway/blob/main/docs/release-notes.md#trino-gateway-5-24-jan-2024)

Reviewer: This needs to move to a separate sentence or so after the 438 section. Its not related to 438

  desc: "437-438"
  time:
---

## Hosts

* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
  at [Starburst](https://starburst.io)
* Manfred Moser, Director of Technical Content at
  [Starburst](https://starburst.io)
  ([@simpligility](https://twitter.com/simpligility))

## Guests

* [Mustafa Sakalsiz](https://linkedin.com/in/sakalsiz), CEO at
  [Peaka](https://www.peaka.com/)
* [Ali Tekin](https://www.linkedin.com/in/alitekin/), Principal Software
  Architect at [Peaka](https://www.peaka.com/)

## Releases 437-438

[Trino 437](https://trino.io/docs/current/release/release-437.html)

* Support for configuring compression codecs
* Support for `char` values in the `to_utf8()` and `lpad()` functions
* Improved performance for Delta Lake queries without table statistics
* Improved performance for Iceberg queries with filters on `ROW` columns

[Trino 438](https://trino.io/docs/current/release/release-438.html)

* Support for access control with Open Policy Agent
* Support for `ALTER COLUMN ... DROP NOT NULL` in Iceberg and PostgreSQL
* Support for configuring page sizes in Delta Lake, Hive, and Iceberg
* Better type support for the `reduce_agg()` function

> 

Reviewer: Additional note here to https://trino.io/blog/2024/02/06/opa-arrived

Reviewer: Also lets add Trino Gateway 5 - https://github.com/trinodb/trino-gateway/blob/main/docs/release-notes.md#trino-gateway-5-24-jan-2024

Reviewer: and charts 016, 0.17 and 0.18 - separate releases but just a few minor improvments

## Concept of the episode: Peaka

Another Trino Community Broadcast episode means another cool piece of technology

> that works with Trino for us to show off to the community. This time it's Peaka,

Reviewer: that uses Trino ?

a no-code approach to date warehousing that makes it easier than ever to set up

> your data stack without needing a ton of complex engineering.

Reviewer: you are not really setting up a stack .. Peaka kinda is the stack .. right? Apart from data sources I guess

Reviewer 2: Yes, Peaka itself is a data stack. The text works for us, very nice.

In [their own words](https://www.peaka.com/docs/getting-started/what-is-peaka/),
Peaka is a platform that merges disparate data sources into a single data layer,
letting you join and blend them, query them using SQL or natural language, and 
expose your data to outside users through APIs. Sounds a bit like Trino, right?
That's because underneath the hood, Trino is a key part of how they're making it
happen. In this episode, we talk to the team at Peaka about where they got
started, how they're making it easier than ever to leverage the federation that
Trino is capable of, and the work they've done on top to integrate their

> platform with ever SaaS data source under the sun.

Reviewer: ```suggestion
platform with every SaaS data source under the sun.
```

Reviewer: also .. many other data sources

## Demo of the episode: Using Peaka!

If you want to see what the platform is like, then look no further. We'll be
exploring:

* Connecting to data sources
* Filtering and combining data
* Editing and running queries, including their visual query editor
* ...and more! (TBD once we have actually streamed the episode)

> ## PR of the episode: #18719: Alluxio cache

Reviewer: Filesystem caching with Alluxio

Perhaps it's a little easier to link to the issue for tracking
[the rollout](https://github.com/trinodb/trino/issues/20550), but however you

> want to present it, caching in Trino is here! Caching is a huge performance win

Reviewer: ```suggestion
want to present it, caching in Trino is renewed! Caching is a huge performance win
```

for a wide variety of use cases, allowing the engine to run faster, better, and

> pump out query results at an unparalleled pace. This is going to lead to huge 

Reviewer: ```suggestion
pump out query results at an unparalleled pace. This is going to lead to 
```

> performance improvements for Trino queries that can leverage it, and you'll hear

Reviewer: ```suggestion
performance improvements for Trino queries using the supported object storage connectors. You'll hear
```
